src/microk8s.py

```diff
@@ -213,3 +213,15 @@ def configure_hostpath_storage(enable: bool):
     else:
         LOG.info("Disable hostpath storage")
         util.ensure_call(["microk8s", "disable", "hostpath-storage"], input=b"n")
+
+
+def configure_rbac(enable: bool):
+    """enable or disable rbac"""
+    LOG.info("Enable RBAC")
```

Comment:
This seems a bit misplaced. First of all, along with the previous log line in charm.py, I might get

> Ensure RBAC mode is false
> Enable RBAC

Which is confusing. I would recommend only keeping a simple `LOG.info()` statement in `microk8s.configure_rbac` instead.

Reply:
not addressed

Reply:
Only logged RBAC here and removed the log statement from config_rbac in charm.py Should I do the reverse?

Reply:
This log statement says nothing about whether we are enabling or disabling RBAC